src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Settings.kt

```diff
@@ -335,19 +339,28 @@ class Settings {
     }
 
     val chat = object {
+        // TODO(BREAKING) Switched to return a string for consistency with the setter
```

Comment:
It seems like the setter should just exclusively use the enum, similar to all of the methods above.